RESPONSIBILITIES.md

```diff
@@ -56,6 +56,17 @@ It's our responsibility to ensure the content and code in pull requests are corr
 - If a pull request is valuable but isn't gaining traction, consider reaching out to fulfill the necessary requirements. This way, the pull request can be merged, even if the work is done by several individuals.
 - Lastly, strive for progress, not perfection.
 
+### Merging a Pull Request 
+
+It is important that commit messages are helpful in understanding the reasons for a given commit and maintain good commit hygiene by only keeping the relevant information. 
+
+When a PR is going to be merged, our repositories are set to automatically squash the commits into a single commit. This process needs human intervention to produce high quality commit messages, with the following steps to be followed as much as possible:
```

Comment:
It's not completely true that all repos force squash, and it's not really correct to say they are "set to squash". They require squashing, mostly. Let's say that:

"Most repositories in opensearch-project are configured to require commits to be squashed when merging pull requests."

I also think we should clarify that the person merging needs to edit the text.

"This process needs human intervention to produce high quality commit messages. When merging contributor pull requests, edit merge messages by following these steps as much as possible."